### ISSUE TITLE
Can I select multiple answers at a time?

### ISSUE THREAD

**Issue by FardinDev** [dmitrizzle/chat-bubble#74]
### Bug or feature request?
Can I select multiple answers at a time?  If not please let me know if there is any way that I can do that.
Will help me and the world a lot

### If it's a bug, please provide the full method to reproduce.
How did you end up seeing the bug? What steps did you take to see it?

**Comment by dmitrizzle** [comment]
You can do it if you build a custom function and a custom interface using this example: https://github.com/dmitrizzle/chat-bubble/blob/master/examples/4-run-scripts.html

You will also need to write an interpreter function to act on your selections. 

Creating a custom interface will involve adding whatever HTML inside the response bubble and also editing your CSS to look decent.

TL;DR: this won't be quick but I think it's possible.
